JPF/utils.py

```diff
@@ -84,11 +84,11 @@ def guess_sqlcol(dfparam, max_col_len):
 
 
 
-def hpd_csv2sql(description, input_csv_url, sep_char,\
-            table_name, dtype_dict, load_pickle, save_pickle, \
-                pickle_file, db_action, truncate_columns, date_time_columns,\
-               sql_chunk_size, keep_cols, max_col_len=255, date_format=None, \
-               csv_chunk_size=None):
+def hpd_csv2sql(description, input_csv_url, sep_char,
+            table_name, dtype_dict, load_pickle, save_pickle,
+                pickle_file, db_action, truncate_columns, date_time_columns,
+               sql_chunk_size, keep_cols, max_col_len=255, date_format=None,
+               csv_chunk_size=None, set_default_primary_key=True):
```

Comment:
I wonder if `False` would be a better default, to stay in line with existing behaviour. I don't think we're actually using the PK anywhere. It could def be useful, but we'd need to collectively figure out a use for it.